apps/www/app/contribute/t/[id]/page.tsx

```diff
@@ -0,0 +1,42 @@
+import Link from 'next/link'
+import { Suspense } from 'react'
+import { ArrowLeft } from 'lucide-react'
+import DefaultLayout from '~/components/Layouts/Default'
+import { ThreadContent } from '~/components/Contribute/ThreadContent'
+import PageLoading from './page-loading'
+import type { Metadata } from 'next'
+import { ContributeGuard } from '../../ContributeGuard'
+
+export const metadata: Metadata = {
+  robots: {
+    index: false,
+    follow: true,
+  },
+}
+
+// eslint-disable-next-line no-restricted-exports
+export default async function ThreadPage({ params }: { params: Promise<{ id: string }> }) {
+  const { id } = await params
+
+  return (
+    <ContributeGuard>
```

Comment:
Added a feature flag for this page so we can get it out without continuing to pile onto this already big PR. Will remove once's it's launched.